### DISCUSSION TITLE
Should GitHub "projects" be migrated from classic?

### DISCUSSION THREAD

**Discussion by EliahKagan** [GitoxideLabs/gitoxide#1390] [Q&A] [answered]
Last week, [Sunset Notice – Projects (classic)](https://github.blog/changelog/2024-05-23-sunset-notice-projects-classic/) was posted on the GitHub Changelog blog.

The "projects" in the gitoxide repository, [Collaboration Board](https://github.com/Byron/gitoxide/projects/1) and [Don't forget for 1.0](https://github.com/Byron/gitoxide/projects/2), are currently [Projects (classic)](https://github.com/Byron/gitoxide/projects?type=classic). Should those be migrated to be the new, non-classic kind of GitHub "project"?

As detailed in that GitHub Changelog post, these will eventually be migrated automatically, but it looks like GitHub is recommending manual migration. I figured it might be good to bring this up. I am not requesting any particular course of action.

**Comment by Byron** [comment, marked as answer]
Thanks for making me aware!

I tried it [with the "Don't forget…" project](https://github.com/users/Byron/projects/5/views/1) and the results are… according to my expectations which weren't high.

Personally, I don't know who designed this supposedly V2 of projects and what they were thinking. V1 was wonderfully complementary to issues, allowing to create free-form cards with markdown and move them around between columns. Trello-light, perfect, very useful.

Now each card has one size, only supports a title to be displayed, and it really wants to be an issue. Markdown is conveniently ignored when converting so one has to fix everything by hand.

Last but not least, the project isn't owned by the repository anymore, but it's just linked there and owned by `Byron` - this feels wrong and is strange to me.

Probably there are many great things about V2 as well, but it's unfortunately not including the functionality of V1 which always made me avoid it :/.

For the off-chance conversion improves till enforced migration, I will wait and see what happens with the remaining project.

**Reply by EliahKagan** [reply]
It occurs to me that it might be possible for GitHub staff to undo the migration.

That wouldn't prevent automatic migration from occurring later, but maybe the feedback would lead to an improvement.

**Reply by Byron** [reply]
It's not that it's terrible, it's just disappointing I suppose, no need to undo.

> That wouldn't prevent automatic migration from occurring later, but maybe the feedback would lead to an improvement.

I am not very familiar with GitHub feedback nor do I have any hope it would change anything. However, if you think differently, I welcome you to provide such feedback. To my mind, Projects V2 is kind of a different product even though technically it could certainly contain V1 as well.
